src/OrbitBase/FileTest.cpp

```diff
@@ -83,6 +83,65 @@ TEST(File, OpenNewFileForReadWrite) {
   ASSERT_TRUE(fd_or_error.has_value()) << fd_or_error.error().message();
 }
 
+TEST(File, WriteFullySmoke) {
+  auto temporary_file_or_error = TemporaryFile::Create();
+  ASSERT_TRUE(temporary_file_or_error.has_value()) << temporary_file_or_error.error().message();
```

Comment:
Shouldn't this be `CHECK` as it's setup (same in `WriteFullyAtOffsetSmoke`)?

Reply:
done